README.md

```diff
@@ -133,6 +133,19 @@ client.stub(
 
 This automatically dumps the body in json and appends the `application/json` Content-Type to the headers.
 
+### Expecting JSON request
+```
+client = MockServerClient("http://localhost:1080")
```

Comment:
could add 

`from mockserver import MockServerClient, request, json_response`

just to be consistent with the rest of the doc